src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -1890,6 +1890,15 @@
   - name: Update a VMSS. (autogenerated)
     text: az vmss update --name MyScaleSet --resource-group MyResourceGroup --set virtualMachineProfile.storageProfile.imageReference.version=16.04.201801090
     crafted: true
+  - name: Update a VMSS' license type for Azure Hybrid Benefit.
```

Comment:
`VMSS's`?

Reply:
I believe if a word ends with 's', you don't add an extra 's' afterwards.